packages/core/admin/admin/src/content-manager/components/AttributeFilter/index.js

```diff
@@ -1,15 +1,80 @@
 import React from 'react';
 
+import { useQueryParams, getDisplayName } from '@strapi/helper-plugin';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 
+import { useAdminUsers } from '../../../hooks/useAdminUsers';
+
+import { AdminUsersFilter } from './AdminUsersFilter';
 import Filters from './Filters';
 import useAllowedAttributes from './hooks/useAllowedAttributes';
 
+const CREATOR_ATTRIBUTES = ['createdBy', 'updatedBy'];
+
 const AttributeFilter = ({ contentType, slug, metadatas }) => {
   const { formatMessage } = useIntl();
+
+  const [{ query }] = useQueryParams();
+  // We get the users selected' ids
+  const selectedUsers =
+    query?.filters?.$and?.reduce((acc, filter) => {
+      const [key, value] = Object.entries(filter)[0];
+      const id = value.id?.$eq || value.id?.$ne;
+
+      if (CREATOR_ATTRIBUTES.includes(key) && !acc.includes(id)) {
+        acc.push(id);
+      }
+
+      return acc;
+    }, []) ?? [];
+  const { users, isLoading } = useAdminUsers(
+    { filter: { id: { in: selectedUsers } } },
+    {
+      enabled: selectedUsers.length > 0,
+    }
+  );
+
   const allowedAttributes = useAllowedAttributes(contentType, slug);
   const displayedFilters = allowedAttributes.map((name) => {
+    if (CREATOR_ATTRIBUTES.includes(name)) {
+      return {
+        name,
+        metadatas: {
+          label: formatMessage({
+            id: `content-manager.components.Filters.${name}`,
+            defaultMessage: name,
+          }),
+          customOperators: [
+            {
+              intlLabel: { id: 'components.FilterOptions.FILTER_TYPES.$eq', defaultMessage: 'is' },
+              value: '$eq',
+            },
+            {
+              intlLabel: {
+                id: 'components.FilterOptions.FILTER_TYPES.$ne',
+                defaultMessage: 'is not',
+              },
+              value: '$ne',
+            },
+          ],
+          customInput: AdminUsersFilter,
+          options: users.map((user) => ({
+            label: getDisplayName(user, formatMessage),
+            customValue: user.id.toString(),
+          })),
+        },
+        fieldSchema: {
+          type: 'relation',
+          mainField: { name: 'id' },
+          trackedEvent: {
+            name: 'didFilterEntries',
+            properties: { useRelation: true },
+          },
+        },
+      };
+    }
```

Comment:
this is very different to other attributes, im sure theres a reason for it – assuming there is can you write a code comment explaining why you're doing this here as opposed to including it as part of `useAllowedAttributes`? :)